superdesk/metadata/item.py

```diff
@@ -220,7 +220,12 @@ class Formats(NamedTuple):
     },
     "subject": {
         "type": "list",
-        "mapping": {"type": "object", "dynamic": False, "properties": {"qcode": not_analyzed, "name": not_analyzed}},
+        "mapping": {
+            "type": "nested",
+            "include_in_parent": True,
```

Comment:
@petrjasek Unfortunately for Analytics we would need nested mapping here, so aggregation on scheme -> qcode/name can be done.

`include_in_parent` is there so non-nested queries for subject still works, but I fear adding nested here will increase the resource requirements within elastic as we're adding a nested doc for each element in this list.

Do you believe this would be OK?

Reply:
I think it's ok, in general in superdesk there should be less content than in eg. newshub with big archive and there we are using it too